packages/flutter_blue/example/pubspec.yaml

```diff
@@ -0,0 +1,21 @@
+name: flutter_blue_tizen_example
+description: Demonstrates how to use the flutter_blue_tizen plugin.
+publish_to: 'none'
+
+environment:
+  flutter: ">=2.0.0"
+  sdk: ">=2.12.0 <3.0.0"
+  
+dependencies:
+  cupertino_icons: ^1.0.2
```

Comment:
Not used in the `flutter_blue` example app: https://github.com/pauldemarco/flutter_blue/blob/master/example/pubspec.yaml
```suggestion
```